### ISSUE TITLE
Improved filter_ABC_primes

### ISSUE THREAD

**Issue by koffie** [isogeny-primes/isogeny-primes#21]
Definition 4.3 and corollary 4.4 give a way to make filter_ABC_primes even better in the case degree > 2. We should implement this at some point since it is quite easy!

**Comment by BarinderBanwait** [comment]
So you mean to implement the notion of a \frak{p}_0-admissible isogeny type? Would this require the choice of such a \frak{p}_0? Because so far no such choice has been made in the code (such a choice does play a role mathematically, but since the code needs to consider all possible epsilon tuples, the choice ends up not making any difference, at least so far).

**Comment by koffie** [comment]
Yeah I'm still thinking a bit about what is the most optimal way to do this. It has to be orchestrated together with #22 I think in order to make sure we have all the dependencies right (since in theory the epsilon type is dependent of the choice of \frak{p}_0.

One way to make sure we do not mess up is looping over all possible choices of \frac{p}_0, and only filtering out p if eps is not admissible for all choices of \frac{p}_0. But it is clear that something is possible here!
